src/styles/getTooltipStyles.js

```diff
@@ -194,6 +194,10 @@ export default function getTooltipStyles(
             fontSize: tooltipFontSize,
             overflow: 'hidden',
             lineHeight: variables.lineHeightSmall,
+
+            // To measure tooltip text width correctly we render it freely i.e. text should not wrap to parent's boundaries.
+            // More info: https://github.com/Expensify/App/issues/15949#issuecomment-1483011998
+            whiteSpace: tooltipContentWidth ? undefined : 'pre',
```

Comment:
```suggestion
            ...(tooltipContentWidth ? {} : whiteSpace.pre),
```

Reply:
Tests well and looks good besides this small change

Reply:
We only use style utilities on the main `styles.js` file. Those utilities are also exported to the main style file so the change you are suggesting should actually be `styles.pre` and not `whiteSpace.pre`. Please update the suggestion